src/scss/froala_blocks.scss

```diff
@@ -18,10 +18,10 @@
   background-color: $white;
 
   &.fdb-viewport {
-    min-height: calc(100% - 2 * $block-padding);
+    min-height: calc(100% - 2 * #{$block-padding});
```

Comment:
It would be better to use arithmetic operations: http://sass-lang.com/documentation/file.SASS_REFERENCE.html#Operations.

Reply:
@stefanneculai thanks for the comment. 

calc() is a string literal in css so we need to interpolate the sass variable, please check SO link below:

https://stackoverflow.com/questions/17982111/sass-variable-in-css-calc-function

and here is the actual sass issue:

https://github.com/sass/sass/issues/818

Thanks!